bin/action_subscriber

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+
+require 'active_support'
+require 'active_support/core_ext'
+require 'thor'
+
+module ActionSubscriber
+  class CLI < ::Thor
+    class_option :app, :default => "./config/environment.rb"
+    class_option :mode
+    class_option :host
+    class_option :hosts
+    class_option :pop_interval, :type => :numeric, :desc => "how long to wait between asking for messages (in milliseconds)"
+    class_option :port, :type => :numeric
+    class_option :threadpool_size, :type => :numeric
+    class_option :times_to_pop, :type => :numeric, :desc => "how many messages to get from each queue each time we ask rabbit"
+
+    desc "start", "Start the babou subscription server"
```

Comment:
These description strings still use `babou` language. Let's change them for consistency.